src/main/java/uk/gov/register/views/ConsistencyProof.java

```diff
@@ -23,7 +26,9 @@ public String getProofIdentifier() {
 
     @SuppressWarnings("unused, used as jsonproperty")
     @JsonProperty("merkle-consistency-nodes")
-    public List<String> getConsistencyNodes() {
+    public List<HashValue> getConsistencyNodes() {
+        List<String> nodes = consistencyNodes.stream().map(p -> p.toString()).collect(Collectors.toList());
```

Comment:
Please also update either `VerifiableLogResourceTest` or `VerifiableLogResourceFunctionalTest` to add a test to make sure the correct prefix is added to the hashes in the `merkle-consistency-nodes`.